core/src/com/unciv/models/ruleset/unique/UniqueType.kt

```diff
@@ -584,6 +584,7 @@ enum class UniqueType(val text: String, vararg targets: UniqueTarget, val flags:
     ConditionalOnWaterMaps("on water maps", UniqueTarget.Conditional),
     ConditionalInRegionOfType("in [regionType] Regions", UniqueTarget.Conditional),
     ConditionalInRegionExceptOfType("in all except [regionType] Regions", UniqueTarget.Conditional),
+    ConditionalFirstCivToResearch("if no other Civilization has researched this", UniqueTarget.Conditional),
```

Comment:
Very minor point, but I think this unique should be put in the `/////// civ conditionals` section instead of the `/////// area conditionals` section of the `uniqueType` enum